common/src/graph.rs

```diff
@@ -16,6 +16,7 @@ use crate::{
 pub struct Graph {
     nodes: FxHashMap<NodeId, NodeContainer>,
     layout: ChunkLayout,
+    horospheres_enabled: bool,
```

Comment:
I realized while setting up this option that we don't actually have a framework for allowing worldgen to be configured in any way. The world is forced to be the same world every time. Since enabling horospheres is a worldgen option, I just had to put it somewhere that works. However, putting it in `Graph` feels wrong. Perhaps fixing this can be punted to another PR, although if we do that, I should probably write a TODO comment here.

Of course, this is meant to be temporary anyway, as scattering random horospheres about is really just meant for debugging megastructure code.

Reply:
Making this configurable at all seems premature, yeah. Might be nice once there's more to tinker with.

Reply:
The main reason I'm making it configurable now is so that people playing on the master branch don't have to deal with horospheres being randomly placed everywhere.

Reply:
I went ahead and made this a constant defined in the horosphere module to simplify things.